utils/kubernetes/pod_service_mapper.py

```diff
@@ -41,7 +50,16 @@ def _fill_services_cache(self):
                 self._service_cache_selectors[uid] = selector
             self._service_cache_invalidated = False
         except Exception as e:
-            log.exception('Unable to read service list from apiserver: %s', e)
+            if isinstance(e, requests.exceptions.HTTPError) and str(e).startswith("403"):
```

Comment:
`str(e).startswith("403")` --> `e.response.status_code == 403` (it's more robust)